### DISCUSSION TITLE
Support for Shelly Pro 4PM

### DISCUSSION THREAD

**Discussion by MichaelVdheeren** [mongoose-os-apps/shelly-homekit#1494] [Ideas]
Any chance we can get support for Shelly4Pro?

**Comment by rojer** [comment]
eventually - sure

**Comment by romanbalsiger** [comment]
into that too

**Comment by RupertBarrow** [comment]
+1

**Comment by MateuszMako** [comment]
+1

**Comment by GilGedje** [comment]
is there any news? really looking forward to use it instead of homebridge

**Comment by Stanly001CZ** [comment]
Hi guys, is there any news about the PM4 or do you know about native support for that model (Apple HomeKit)? Thank you so much

**Comment by jakubStep** [comment]
Hi, I would love to have support for any of DIN rail mountable Shelly Pro switches. Thank you in advance!

**Comment by psa2405** [comment]
Any news about the Pro 4PM? A HomeKit software would be highly appreciated.

**Comment by jhulbertpmn** [comment]
+1 (any of the pro din mounted devices)

**Comment by Borne90** [comment]
+1
Do the Pro Devices have a different structure than the  "normal" PM ones which would make it harder to convert/support?

**Comment by pterhaar** [comment]
yes please. All the Pro devices please, I have quite a few of them
